frontend/components/Main/MainHeader.tsx

```diff
@@ -122,7 +177,14 @@ export const MainHeader = () => {
       })
         .then(() => {
           setServiceStatus(DeploymentStatus.DEPLOYED);
-          showNotification?.('Your agent is now running!');
+          if (totalOlasStakedBalance === 0) {
```

Comment:
not sure if this is correct; we're check if it's 0 and saying they've staked 0.

Reply:
at that time this is correct, because the previous value was 0. do you think it worths adding a comment?

Reply:
updated @truemiller lmk if it's clearer